### PR TITLE
docs: add auth tutorial example

### PR DESCRIPTION
**Pull request by jannyHou**
Add example for https://github.com/strongloop/loopback-next/issues/5573

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Authentication-overview.md [hunk 1]
```diff
@@ -68,3 +72,7 @@ we've broken down the documentations into several parts:
   Particularly for extension developers.
 - [Use **Express Passport** strategies](Authentication-passport.md): The usage
   of passport adapter module `@loopback/authentication-passport`.
+
+## Examples
+
+There is a list of authentication related examples created by LoopBack team or contributed by community members. They can be found in [authentication examples](https://loopback.io/doc/en/lb4/Examples.html#authentication-and-authorization-related-examples).
```
Comment (nabdelgadir) [inline docs/site/Authentication-overview.md:78]: Nitpick: can you use the relative path for the link?

File: docs/site/Examples.md [hunk 2]
```diff
@@ -64,6 +64,9 @@ LoopBack 4 comes with the following example projects.
   [Todo example](https://github.com/strongloop/loopback-next/tree/master/examples/todo)
   with JWT authentication.
 
+- **[build jwt authentication from scratch](https://github.com/HrithikMittal/Loopback4-auth)**:
```
Comment (nabdelgadir) [inline docs/site/Examples.md:67]: Just wondering if we should mention this is a community example? I know you mentioned in the other page that some are community examples, but if someone goes directly to this page they wouldn't see that note.

File: docs/site/Authentication-overview.md (hunk 1 above)
Comment (hacksparrow) [reply docs/site/Authentication-overview.md:78]: Yes, to be consistent with other pages.

File: docs/site/Examples.md (hunk 2 above)
Comment (dhmlau) [reply docs/site/Examples.md:67]: If we're planning to have more community examples on this page, perhaps we can use superscript or asterisk to indicate this is community example?

Comment (jannyHou) [reply docs/site/Examples.md:67]: I realized this example page is for LB supported examples only...let me find move it to another place.

Comment (jannyHou) [reply docs/site/Examples.md]: I moved them back to be under the auth overview page. And use `(community)` to mark community examples.

**Pull request merged**